packages/camera_driver/src/camera_node.py

```diff
@@ -8,15 +8,19 @@
 from threading import Thread
 
 # for Jetson Nano
-import cv2
-import atexit
-import threading
-from cv_bridge import CvBridge, CvBridgeError
+if (os.environ.get('ROBOT_HARDWARE') == "jetson_nano"):
+    import cv2
+    import atexit
+    import threading
+    from cv_bridge import CvBridge, CvBridgeError
 
 # for RPi
-if (os.environ.get('ROBOT_HARDWARE') == "raspberry_pi"):
+elif (os.environ.get('ROBOT_HARDWARE') == "raspberry_pi"):
     from picamera import PiCamera
 
+else:
+    raise Exception("Undefined Hardware!")
```

Comment:
"Undefined Hardware!" -> "Undefined Hardware! The environment variable ROBOT_HARDWARE must be set!"